drivers/linstorvolumemanager.py

```diff
@@ -698,6 +700,13 @@ def destroy_volume(self, volume_uuid):
         self._ensure_volume_exists(volume_uuid)
         self.ensure_volume_is_not_locked(volume_uuid)
 
+        is_volume_in_use = any(node["in-use"] for node in self.get_resource_info(volume_uuid)["nodes"].values())
```

Comment:
Honestly, I don't remember exactly what we said, but: Honestly, I don't remember exactly what we said, but: how many seconds do you estimate the cost of this call to be? For example, if we have 500 resources in the database?